java/openjdk/Portfile

```diff
@@ -472,7 +472,11 @@ subport openjdk16-openj9 {
 }
 
 subport openjdk16-zulu {
-    version      16.30.15
+    if {${configure.build_arch} eq "x86_64"} {
+        version      16.30.15
+    } elseif {${configure.build_arch} eq "arm64"} {
+        version      16.30.19
```

Comment:
Why not update for intel as well ?

Reply:
Do you know if its likely to come soon, or in general is a different version for each going to be something they do going forward ?

Reply:
I think they might have done an extra release for arm64, because I got a report that 16.30.15 doesn't work on arm64.

Reply:
I don't have an arm64 machine myself though, so sadly I can't verify this.